pkgs/stacklok/frizbee/registry.yaml

```diff
@@ -16,16 +16,49 @@ packages:
         overrides:
           - goos: windows
             format: zip
-      - version_constraint: "true"
+      - version_constraint: Version == "v0.1.5"
+        asset: frizbee_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        checksum:
+          type: github_release
+          asset: frizbee_{{trimV .Version}}_checksums.txt
+          algorithm: sha256
+        overrides:
+          - goos: windows
+            format: zip
+      - version_constraint: Version == "v0.1.6"
         asset: frizbee_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
         format: tar.gz
+        checksum:
+          type: github_release
+          asset: frizbee_{{trimV .Version}}_checksums.txt
+          algorithm: sha256
         slsa_provenance:
           type: github_release
           asset: multiple.intoto.jsonl
+        cosign:
+          opts:
+            - --certificate
+            - https://github.com/stacklok/frizbee/releases/download/{{.Version}}/{{trimAssetExt .Asset}}.pem
```

Comment:
This does not obviously currently work, but added here for illustration purposes, see https://github.com/aquaproj/aqua/pull/3665